disperser/batcher/minibatch_store.go

```diff
@@ -0,0 +1,55 @@
+package batcher
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/core"
+	gcommon "github.com/ethereum/go-ethereum/common"
+)
+
+type BatchID time.Time
```

Comment:
I think the better option would be to go with UUIDv7 https://pkg.go.dev/github.com/google/uuid#NewV7

UUIDv7 gives us sorting and entropy to avoid collisions. Time.Now() works, but the lack of entropy makes collisions a real possibility if we ever introduce multi-threading or distributed minibatchers

Reply:
I guess one benefit of using nanoseconds is that the primary key field is smaller vs UUID. After reading dynamo docs some more, using timestamp as primary key is pretty common pattern. So now I think time.Time is fine.